lib/game.rb

```diff
@@ -0,0 +1,147 @@
+class Game
+
+  attr_accessor :board, :player_1, :player_2
+
+  WIN_COMBINATIONS = [
+    [0,1,2],  #0
+    [3,4,5],  #1
+    [6,7,8],  #2
+    [0,3,6],  #3
+    [1,4,7],  #4
+    [2,5,8],  #5
+    [0,4,8],  #6
+    [2,4,6]   #7
+  ]
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+    @board = board
+    @player_1 = player_1
+    @player_2 = player_2
+  end
+
+  def current_player
+    if board.turn_count % 2 == 0
+      @player_1
+    elsif board.turn_count == 1
+      @player_2
+    elsif board.turn_count == 0
+      @player_1
+    elsif board.turn_count % 2 != 0
+      @player_2
+    end
+  end
+
+  def won?
```

Comment:
Consider using [`Enumerable#detect`](http://ruby-doc.org/core-2.2.3/Enumerable.html#method-i-detect) to return the winning combination or `nil` if none is found.

```ruby
  def won?
    WIN_COMBINATIONS.detect do |combo|
      @board.cells[combo[0]] == @board.cells[combo[1]] &&
      @board.cells[combo[0]] == @board.cells[combo[2]] &&
      @board.taken?(combo[0] + 1)
    end
  end
```